docs/static/configuration.asciidoc

```diff
@@ -188,6 +188,8 @@ match => {
   "field2" => "value2"
   ...
 }
+# or
+match => { "field1" => "value1" "field2" => "value2" }
```

Comment:
Hmm, what about spreading this on multiple lines to (hopefully) improve readability?

Reply:
The purpose of this addition is to clearly show how to do single line hashes. It gets people confused.

I agree that multi-line is clearer. But sometimes you just have 2 key-values, and the syntax that looks like Ruby hashes confuses people. They wonder why `{ "field1" => "value1", "field2" => "value2" }` doesn't work.

That's also why I put the single line example just below the multi-line example. To convey that multi-line should be the first option.

Reply:
Ahh thanks for the explanation. My preference to avoid this in-line style blinded me to your goal.

Small suggestion, change `# or` to `# or as a single line. No commas between entries:`